syft/pkg/cataloger/dotnet/deps_binary_embed_deb_test.go

```diff
@@ -0,0 +1,21 @@
+package dotnet
+
+import (
+	"testing"
+
+	"github.com/anchore/syft/syft/pkg/cataloger/internal/pkgtest"
+)
+
+func Test_EmbeddedDepsExtraction(t *testing.T) {
```

Comment:
the checked in binary is not complete (not a valid PE file) yet we still pick up on deps.json resources. This is probably ok, but not ideal.